torch/_dynamo/guards.py

```diff
@@ -2300,17 +2296,17 @@ def add_code_part(code_part, guard, log_only=False):
                     add_code_part(code, gcl.guard, True)
                     seen.add(code)
 
-        tensor_check_names = builder.tensor_check_names
```

Comment:
Does the code in torch/csrc/dynamo/guards.cpp referencing tensor_check_names have to be adjusted?

Also, since this happened before, can you explicitly make sure our verbose reporting (e.g., in logs and tlparse) did not regress with these changes? (The switch to cpp guard manager caused us to lose all tlparse guard output)